api/turing/imagebuilder/imagebuilder.go

```diff
@@ -288,12 +293,23 @@ func (ib *imageBuilder) createKanikoJob(
 		fmt.Sprintf("--context=%s", ib.imageBuildingConfig.KanikoConfig.BuildContextURI),
 		fmt.Sprintf("--build-arg=MODEL_URL=%s", artifactURI),
 		fmt.Sprintf("--build-arg=BASE_IMAGE=%s", baseImage),
+		fmt.Sprintf("--build-arg=MLFLOW_ARTIFACT_STORAGE_TYPE=%s", ib.artifactServiceType),
 		fmt.Sprintf("--build-arg=FOLDER_NAME=%s", folderName),
 		fmt.Sprintf("--destination=%s", imageRef),
 		"--cache=true",
 		"--single-snapshot",
 	}
 
+	if ib.artifactServiceType == "s3" {
+		kanikoArgs = append(
```

Comment:
can we read it from environment files?

Reply:
Okay I've done this like in Merlin [here](https://github.com/caraml-dev/merlin/pull/605#discussion_r1807766235) where there's a config field in the config value files where platform-level additional kaniko args can be added, instead of using an env file to keep things consistent. If you think an env file is cleaner we can change it to that too.